magnet/excitation-data/si-corrector-ivu18-ch.txt

```diff
@@ -8,9 +8,9 @@
 
 # EXCITATION DATA
 # ===============
--0001.0000  -1.8000e-04 +0.0000e+00
+-0002.0000  -1.0277e-04 +0.0000e+00
 +0000.0000  +0.0000e+00 +0.0000e+00
-+0001.0000  +1.8000e-04 +0.0000e+00
++0002.0000  +1.0277e-04 +0.0000e+00
 
 # COMMENTS
 # ========
```

Comment:
@Gabrielrezende-asc , these comments should be updated. I believe you used LNLS measured data, right? we should add link to the analysis doc ou measured data.